integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md

```diff
@@ -127,11 +130,12 @@ Afterward, you will have a model that achieves roughly 28.7 mAP on the COCO data
     │   ├── dets
     │   └── scripts
     └── weights
+        ├── darknet53.pth
         └── yolact_darknet53_54_800000.pth
  ```
 
 You are ready to use the weights at `yolact/weights/yolact_darknet53_54_800000.pth` with the Neural Magic recipes to create a sparsified model.
-
+You can also download directly this pre-trained weight from [here](https://drive.google.com/file/d/1dukLrTzZQEuhzitGkHaGjphlmRJOjVnP/view)
```

Comment:
@rahul-tuli these should be the base weights on the SparseZoo for fp32, please update accordingly rather than going through google drive

Reply:
```suggestion
You can also download this baseline, pre-trained checkpoint directly from the [SparseZoo UI](https://staging-sparsezoo.neuralmagic.com/models/cv%2Fsegmentation%2Fyolact-darknet53%2Fpytorch%2Fdbolya%2Fcoco%2Fbase-none).
```